tests/integration/utils/link_process.go

```diff
@@ -44,11 +44,11 @@ func StartLinK(t *testing.T, binaryPath string, opts ...StartLinkOpt) LinKProces
 
 	cmd := exec.Command(binaryPath)
 	cmd.Env = append(cmd.Env, fmt.Sprintf("PORT=%d", port))
-	cmd.Env = append(cmd.Env, "ETCD_HOSTS=http://localhost:2379")
 	cmd.Env = append(cmd.Env, "INTERFACE=eth0")
 	cmd.Env = append(cmd.Env, "HOSTNAME=test-host")
 	cmd.Env = append(cmd.Env, "KEEPALIVE_INTERVAL=100ms")
-	cmd.Env = append(cmd.Env, "GO_ENV=test")
+	cmd.Env = append(cmd.Env, "ETCD_HOSTS="+os.Getenv("ETCD_HOSTS"))
+	cmd.Env = append(cmd.Env, "GO_ENV="+os.Getenv("GO_ENV"))
```

Comment:
Both these environment variables are defined in the environment. We want to use the value that is defined in the environment. In the CI and in dev, the `ETCD_HOSTS` value is not the same.